plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -910,6 +912,255 @@ public void testSettingChangeDataFeedEnabledProperty()
                 .contains("change_data_feed_enabled = true");
     }
 
+    @Test
+    public void testReadingChangesMadeByBasicOperationsOnTableWithCdfEnabled()
+    {
+        String tableName = "test_basic_operations_on_table_with_cdf_enabled_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " (page_url VARCHAR, domain VARCHAR, views INTEGER) WITH (change_data_feed_enabled = true)");
+        assertUpdate("INSERT INTO " + tableName + " VALUES('url1', 'domain1', 1), ('url2', 'domain2', 2), ('url3', 'domain3', 3)", 3);
+        assertUpdate("INSERT INTO " + tableName + " VALUES('url4', 'domain4', 4), ('url5', 'domain5', 2), ('url6', 'domain6', 6)", 3);
+
+        assertUpdate("UPDATE " + tableName + " SET page_url = 'url22' WHERE views = 2", 2);
+        MaterializedResult result = computeActual("SELECT * FROM TABLE(system.table_changes('test_schema', '" + tableName + "'))");
```

Comment:
I was wondering what columns are exposed by the table function, but ran into a parsing exception

```
computeActual("SHOW COLUMNS FROM TABLE(system.table_changes('test_schema', '" + tableName + "'))")
```

Is it foreseen that this kind of query will be eventually supported?

cc @kasiafi  @martint